test/unit/core/action-creator.ts

```diff
@@ -147,6 +147,7 @@ suite('ActionCreator', ({ expect, spy, stub }) => {
         const search: any = { a: 'b' };
         const state = { a: 'b' };
         stub(Selectors, 'query').withArgs(state).returns(query);
+        stub(Selectors, 'selectedRefinements').withArgs(state).returns([]);
```

Comment:
don't need to specify `withArgs()` here, we should only use this when we care about returning multiple distinct results based on the input, but as `selectedRefinements()` is only called once.
Instead it would be easier to include a test that explicitly expects `selectedRefinements` to be called with the `state`, that way we don't need to include it in all the other tests

```js
const selectedRefinements = stub(Selectors, 'selectedRefinements').returns([]);

// ...

expect(selectedRefinements).to.be.calledWith(state)`
```